inference-engine/tests/functional/plugin/gna/shared_tests_instances/behavior/core_integration.cpp

```diff
@@ -76,49 +76,57 @@ INSTANTIATE_TEST_CASE_P(
 // Executable Network GetMetric
 //
 
+// TODO: 2D convolution is not supported by GNA
 INSTANTIATE_TEST_CASE_P(
-        IEClassExecutableNetworkGetMetricTest, IEClassExecutableNetworkGetMetricTest_SUPPORTED_CONFIG_KEYS,
```

Comment:
@iefode Can we use `DISABLED` macro here or we should use `SKIP_IF_CURRENT_TEST_IS_DISABLED()` ?

Reply:
IMO, `skip_test_config` is more preferable, because you will have all bugs in single place. But the GNA plugin team will maintain these tests and you should deсide which way is more comfortable

Reply:
Ok. So let's keep it as is right now and change it next PR.